app/controllers/api/v2/webhooks_controller.rb

```diff
@@ -23,7 +23,9 @@ def show; end
         param :webhook, Hash, action_aware: true, required: true do
           param :name, String, required: true
           param :target_url, String, required: true
-          param :events, Array, required: true
+          param :http_method, Webhook.allowed_http_methods
+          param :http_content_type, String
+          param :event, String, required: true
```

Comment:
Why did you change `events` to `event`?

Reply:
I thought we are not changing this in this PR.